client/src/views/Chat.vue

```diff
@@ -329,6 +328,17 @@ export default {
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+*{
+  font-family: "Roboto","Poppins", "Helvetica", sans-serif;
+}
```

Comment:
```suggestion
@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');

*{
  font-family: "Roboto", "Helvetica", sans-serif;
}
```

Reply:
This is the final change required to get this merged.

Reply:
@PradyumnaKrishna Done Sir _/\_